mailserver/mailserver.go

```diff
@@ -248,46 +250,31 @@ func (s *WMailServer) validateRequest(peerID []byte, request *whisper.Envelope)
 	return true, lower, upper, bloom
 }
 
-type limiter struct {
-	mu sync.RWMutex
-
-	timeout time.Duration
-	db      map[string]time.Time
-}
-
-func newLimiter(timeout time.Duration) *limiter {
-	return &limiter{
-		timeout: timeout,
-		db:      make(map[string]time.Time),
+func (s *WMailServer) checkMsgSignature(msg *whisper.ReceivedMessage, id []byte) error {
```

Comment:
Much cleaner with the extra functions. 👍 But also always a friend of comments even for private functions. I.e. semantical details about the check here. 😉

Reply:
yep, I was being a bit lazy with private comments, i'll add them now :)